src/icons.jsx

```diff
@@ -9,6 +9,7 @@ const importIcons = function () {
     'icon-album-remove',
     'icon-back',
     'icon-check',
+    'icon-check-circleless',
```

Comment:
Mmh, should have been the otherway around :/

icon-check and icon-check-circle

Reply:
Exactly :/ Hopefully we can do this when we next break backward compatibility.